owncloudApp/src/main/java/com/owncloud/android/presentation/files/createfolder/CreateFolderDialogFragment.kt

```diff
@@ -107,6 +148,7 @@ class CreateFolderDialogFragment : DialogFragment() {
     companion object {
         const val CREATE_FOLDER_FRAGMENT = "CREATE_FOLDER_FRAGMENT"
         private const val IS_BUTTON_ENABLED_FLAG_KEY = "IS_BUTTON_ENABLED_FLAG_KEY"
+        private val forbiddenChars = listOf('/', '\\')
```

Comment:
I prefer this approach rather than the regex one! In this case it's a simple case and I think it's better understood this way 🥇